src/util.ts

```diff
@@ -8,10 +8,11 @@
 // tslint:disable-next-line:no-unbound-method
 const toString = Object.prototype.toString
 
-/** Reports whether value is a function. */
+/** Reports whether value is a function (or an async function). */
 // tslint:disable-next-line:ban-types
 export function isFunction(value: any): value is Function {
```

Comment:
What's the benefit of this function over `typeof value === 'function'`?

Reply:
None anymore. Will change.